fe/src/pages/campaigns/[id].vue

```diff
@@ -1,14 +1,49 @@
 <template>
   <div
-    class="flex flex-col gap-6 py-6 px-36 flex-1 bg-linear-gradient-white-to-light overflow-auto"
+    v-if="isPageLoading"
+    class="flex justify-center items-center w-full h-screen"
+  >
+    <div class="flex justify-center items-center">
+      <div
+        class="animate-spin rounded-full h-8 w-8 border-b-2 border-gray-900"
+      ></div>
+    </div>
+  </div>
```

Comment:
Can you make this into a reusable component since it is used many times?
![image](https://github.com/framgia/sph-solidity/assets/57890826/ef9198ea-b10e-442b-8f7a-15d3a66667b5)

Reply:
Sure, Thanks! 🙇‍♂️